Users and Groups APIs supports the contains operator (#4416)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

| `ne`       | not equal             | If the attribute value does not match the operand value, there is a match. |
| `pr`       | present (has value)   | If the attribute has a non-empty value or if it contains a non-empty node for complex attributes, there is a match.                                                                                                                                                           |
| `sw`       | starts with           | The entire operand value must be a substring of the attribute value that starts at the beginning of the attribute value. This criterion is satisfied if the two strings are identical.                                                                                         |

> | `co`       | contains              | If the operand value exists as a substring in the attribute value, there's a match. This includes the case where the attribute and operand values are identical.                       |

Reviewer: I'm hesitant to add this operator in the table ... because it's only used by Systems Log, and now Users and Groups. I looked into adding this operator to this table about a month and a half ago and it was decided just to add the note below. I'd need to look into the ticket, but I believe this was an engineering decision, not mine.

| `ne`       | not equal             | If the attribute value does not match the operand value, there is a match. |
| `pr`       | present (has value)   | If the attribute has a non-empty value or if it contains a non-empty node for complex attributes, there is a match.                                                                                                                                                           |
| `sw`       | starts with           | The entire operand value must be a substring of the attribute value that starts at the beginning of the attribute value. This criterion is satisfied if the two strings are identical.                                                                                         |
| `ew`       | ends with             | The entire operand value must be a substring of the attribute value that starts at the end of the attribute value. This criterion is satisfied if the two strings are identical. This operator is only usable with the System Log API.                          |

> | `co`       | contains              | If the operand value exists as a substring in the attribute value, there's a match. This includes the case where the attribute and operand values are identical. This operator is only usable with the [Groups](/docs/reference/api/groups/#list-groups-with-search), [System Log](/docs/reference/api/system-log/#filtering-results), and [Users](/docs/reference/api/users/#list-users-with-search) APIs.                      |

Reviewer: ```suggestion
| `co`       | contains              | If the operand value exists as a substring in the attribute value, there's a match. This includes the case where the attribute and operand values are identical. You can only use this operator with the [Groups](/docs/reference/api/groups/#list-groups-with-search), [System Log](/docs/reference/api/system-log/#filtering-results), and [Users](/docs/reference/api/users/#list-users-with-search) APIs.                      |
```
